### PR TITLE
refactor(navigation): implement left navigation new specs

### PR DESCRIPTION
**Pull request by badejayayesubabu**
## Description

`<navigation>` : implement left navigation new fabric styles

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: Navigation-SURF-2172

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: docs/components/navigation/index.html [hunk 1]
```diff
@@ -35,27 +35,17 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
...
-              <a href="#">Link 3-3</a>
-            </hx-reveal>
           </hx-reveal>
         </nav>
       </div>
 
       <footer>
         {% code 'html' %}
-          <nav class="hxNav">
+          <nav class="hxNav sub-leftNav">
```
Comment (100stacks) [inline docs/components/navigation/index.html:48]: Please modify to add a `hx` prefix to any Helix classes we add:

### Please change:

```css
sub-leftNav
```

### To:

```css
hxLeftNavSection
```

File: src/scss/components/navigation/_index.scss [hunk 2]
```diff
@@ -93,3 +100,37 @@
     }
   }//hx-reveal
 }
+
+// ===== SUB-LEFT NAVIGATION =====
+.sub-leftNav, .first-section {
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:105]: Please modify to add a `hx` prefix to any Helix classes we add.

### Please change:

```scss
.sub-leftNav
```

### To:

```scss
.hxLeftNavSection
```

File: src/scss/components/navigation/_index.scss [hunk 3]
```diff
@@ -93,3 +100,37 @@
...
+  > a:active, > a:focus {
+    border-left: 0.125rem solid $purple-500;
+    color: $purple-500;
+    font-weight: 600;
+    outline: none;
+  }
+}
+
+.sub-leftNav a:active, .sub-leftNav a:focus {
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:114]: ### Please change:

```scss
.sub-leftNav
```

### To:

```scss
.hxLeftNavSection
```

File: src/scss/components/navigation/_index.scss (hunk 2 above)
Comment (100stacks) [reply src/scss/components/navigation/_index.scss:105]: ### Please change:

```css
.first-section
```

To:

```css
.hxLeftNavSubSection
```

File: src/scss/components/navigation/_index.scss [hunk 4]
```diff
@@ -93,3 +100,37 @@
...
+  margin-left: 0.3rem;
+  padding-left: 0.8rem;
+}
+
+nav .first-section {
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:119]: ### Please change:

```css
.first-section
```

To:

```css
.hxLeftNavSubSection
```

File: src/scss/components/navigation/_index.scss [hunk 5]
```diff
@@ -93,3 +100,37 @@
...
+  > a:active, > a:focus {
+    margin-left: 1.25rem;
+    padding-left: 1.125rem;
+  }
+}
+
+nav hx-reveal.first-section {
```
Comment (100stacks) [inline src/scss/components/navigation/_index.scss:126]: ### Please change:

```css
.first-section
```

To:

```css
.hxLeftNavSubSection
```

File: docs/components/navigation/index.html [hunk 6]
```diff
@@ -64,20 +54,10 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
               L1 Section
               <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
             </hx-disclosure>
-            <hx-reveal id="section-1" open>
+            <hx-reveal class="first-section" id="section-1" open>
```
Comment (100stacks) [inline docs/components/navigation/index.html:57]: ### Please change:

```css
first-section
```

To:

```css
hxLeftNavSubSection
```

File: docs/components/navigation/index.html [hunk 7]
```diff
@@ -26,7 +26,7 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
 
     <div class="example">
       <div class="nav-demo">
-        <nav class="hxNav">
+        <nav class="hxNav sub-leftNav">
```
Comment (100stacks) [inline docs/components/navigation/index.html:29]: Please see comments below.

File: docs/components/navigation/index.html [hunk 8]
```diff
@@ -35,27 +35,17 @@ <h2 id="left-navigation">Left (Vertical) Navigation</h2>
             L1 Section
             <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
           </hx-disclosure>
-          <hx-reveal id="section-1" open>
+          <hx-reveal class="first-section" id="section-1" open>
```
Comment (100stacks) [inline docs/components/navigation/index.html:38]: Please see comments below.

**Review by 100stacks — CHANGES_REQUESTED** [review]
### Sorry for the delay...please see minor tweaks.

File: docs/components/navigation/index.html (hunk 1 above)
Comment (badejayayesubabu) [reply docs/components/navigation/index.html]: Changes have been done, Please review

File: src/scss/components/navigation/_index.scss (hunk 2 above)
Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Changes have been done, Please review

File: src/scss/components/navigation/_index.scss (hunk 3 above)
Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Changes have been done, Please review

File: src/scss/components/navigation/_index.scss (hunk 4 above)
Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Changes have been done, Please review

File: src/scss/components/navigation/_index.scss (hunk 5 above)
Comment (badejayayesubabu) [reply src/scss/components/navigation/_index.scss]: Changes have been done, Please review

File: docs/components/navigation/index.html (hunk 6 above)
Comment (badejayayesubabu) [reply docs/components/navigation/index.html]: Changes have been done, Please review

File: docs/components/navigation/index.html (hunk 7 above)
Comment (badejayayesubabu) [reply docs/components/navigation/index.html]: Changes have been done, Please review

File: docs/components/navigation/index.html (hunk 8 above)
Comment (badejayayesubabu) [reply docs/components/navigation/index.html]: Changes have been done, Please review

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Review by 100stacks — APPROVED** [review]
### Great work @badejayayesubabu!

Dev LGTM

**Pull request merged**